next/components/forms/widget-wrappers/UploadWidgetRJSF.tsx

```diff
@@ -93,6 +94,7 @@ const UploadWidgetRJSF = (props: UploadWidgetRJSFProps) => {
         sizeLimit={size}
         supportedFormats={supportedFormats}
         disabled={disabled || readonly}
+        additionalLinks={additionalLinks}
```

Comment:
Maybe WidgetWrapper is more suitable even?

Reply:
you are right - moved there and added to every widget-wrapped element